browser/js/account/accountFactory.js

```diff
@@ -0,0 +1,26 @@
+app.factory('AccountFactory', function($http,AuthService){
+	return {
+		getAccountInformation: function(){
```

Comment:
These routes don't comply with the put routes already defined in the user index.js file. Please check and re factor. thanks